stratum/hal/lib/bmv2/bmv2_chassis_manager.cc

```diff
@@ -521,9 +523,9 @@ ::util::Status Bmv2ChassisManager::Shutdown() {
   // It is fine to release the chassis lock here (it is actually needed to call
   // UnregisterEventWriters or there would be a deadlock). Because initialized_
   // is set to true, RegisterEventWriters cannot be called.
-  APPEND_STATUS_IF_ERROR(status, UnregisterEventWriters());
```

Comment:
Should we look at the comment above?

Also, if we do decide to keep the lock to unregister the event writer, then we should probably just merge this with the block above.

Reply:
mhh, good catch.

I'll have to the verify the lock usage again.